Bottom: 76691c2341e7f5baa434d8c446c14e2fccb01402
Top:    d1f0a9d8b35486dc7efc35fe7ab25b5844836117
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 02:03:29 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
